### PR TITLE
fix: 🐛 Allow `modifyClaims` to modify claim with cdd_id `0x00`

### PR DESCRIPTION
**Pull request by prashantasdeveloper**

### CODE REVIEW

**Review by polymath-eric — APPROVED** [review]

**Review by VictorVicente — CHANGES_REQUESTED** [review]
@prashantasdeveloper I left a comment regarding CddIds.
Also, could you check we are covering all these cases with unit tests:
1) `currentCddId` is the default and `newCddId` is different (should success)
2) `newCddId` is the default and `currentCddId` is different (should success)
3) `currentCddId` is some_value and `newCddId` is the another_value (should be invalid)

File: src/utils/constants.ts [hunk 1]
```diff
@@ -134,3 +134,8 @@ export const SYSTEM_VERSION_RPC_CALL = {
   method: 'system_version',
   params: [],
 };
+
+/**
+ * Default cdd_id associated with an Identity on chain
```
Comment (monitz87) [inline src/utils/constants.ts:139]:
```suggestion
 * Default CDD ID associated with an Identity on chain. Used for Identities onboarded without PUIS
```

**Review by monitz87 — APPROVED** [review]

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**